spec/support/login_helper.rb

```diff
@@ -1,15 +1,18 @@
-def sign_in
-  set_omniauth
-  visit "/"
+def log_in(user)
+  OmniAuth.config.test_mode = true
+  OmniAuth.config.mock_auth[:google_oauth2] = OmniAuth::AuthHash.new(
+  {
+    provider: user.provider,
+    uid: user.uid,
+    info: {
+      name: user.name,
+      email: user.email
+    },
+    credentials: {
+      token: user.oauth_token,
+      expires_at: user.oauth_expires_at
+    }
+    })
```

Comment:
Indent the right brace the same as the start of the line where the left brace is.